src/main/proto/wfa/measurement/api/v2alpha/data_provider.proto

```diff
@@ -74,12 +74,12 @@ message DataProviderPublicKey {
   // Resource key.
   DataProvider.Key key = 1;
 
-  // Encryption public key for this `DataProvider`, which can be verified using
-  // `data_provider_certificate`. Required.
+  // Serialized `EncryptionPublicKey` for this `DataProvider`, which can be
+  // verified using `data_provider_certificate`. Required.
```

Comment:
This one rotates while the Certificate.key doesn't, correct? Does that change how we want to structure this?

Reply:
why not just call the field `encryption_public_key`?

Reply:
I think something went wrong in merging -- this definitely shouldn't be part of this PR. This is from a PR that Sanjay already landed.